giotto/homology/tests/test_grids.py

```diff
@@ -0,0 +1,30 @@
+"""Testing for persistent homology on grids."""
+# License : Apache 2.0
```

Comment:
I think we don't need to inject the license on each file - isn't this covered by the repo's license?

Reply:
It is. Hold habit from the time when authors and license where written on the top of the file.
I will remove it.

Reply:
@ulupo can you please advise what the current practice is re the license (see #139)?

Reply:
It was also written mimicking [VietorisRipsPersisentence](https://github.com/giotto-ai/giotto-learn/blob/master/giotto/homology/point_clouds.py#L1) but I dont' mind making the change.

Reply:
I wrote a relevant comment in (#139). We'll make the decision later.  Since these comments are currently there everywhere else, @gtauzin could you add "# License: Apache 2.0" to each file in this PR save for `__init__.py` -- in particular, could you change "License :" to "License: " in this particular file?

Reply:
Done.